matching-service/src/services/rabbitMqService.ts

```diff
@@ -0,0 +1,59 @@
+import amqp, { Channel } from "amqplib";
+import { storeUser } from "../model/userModel";
+import { User } from "../types";
+import { processNewUser } from "./matchingService";
+
+let channel: Channel;
+const rabbit_url = process.env.RABBITMQ_URL || "amqp://localhost";
+
+export const initRabbitMQ = async (queue: string): Promise<void> => {
+  try {
+    const connection = await amqp.connect(rabbit_url);
```

Comment:
I think connection can be made in init, then the rest of the consume functionality below can be separated into receiveFromQueue function liek what you did for the sendToQueue below